### PR TITLE
[test-only] api tests for getting group info of user

### PR DESCRIPTION
**Pull request by ScharfViktor**
I added test for getting group info of the user: `curl -X GET 'https://localhost:9200/graph/v1.0/users/2569c4ad-4abc-4f40-9490-5db0b66dcedc?%24expand=memberOf' -uadmin:admin -vk`

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiGraph-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/18597/44/1) failed. Further test are cancelled...

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 1]
```diff
@@ -75,3 +75,15 @@ Feature: get users
       | root@@@id         | %space_id%                       |
       | root@@@webDavUrl  | %base_url%/dav/spaces/%space_id% |
       | webUrl            | %base_url%/f/%space_id%          |
+
+
+  Scenario: admin user gets the group information of a user
```
Comment (SagarGi) [inline tests/acceptance/features/apiGraph/getUser.feature:80]: @ScharfViktor can we also add a scenario like `non-admin user gets the group information of another user` if feasible.

**Review by SagarGi — COMMENTED** [review]
Others LGTM :+1:

File: tests/acceptance/features/apiGraph/getUser.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/apiGraph/getUser.feature:80]: Yes, true. Thanks)

**Review by SagarGi — APPROVED** [review]

**Pull request merged**